pkg/apis/ark/v1/backup.go

```diff
@@ -109,6 +109,10 @@ type VolumeBackupInfo struct {
 	// API.
 	Type string `json:"type"`
 
+	// AvailabilityZone is the where the volume is provisioned
+	// in the cloud provider.
+	AvailabilityZone string `json:"avaialabiltyZone"`
```

Comment:
small typo in the json tag -- should be `json:"avaialabiltyZone"`

Reply:
Thanks for catching that!